src/styles/index.scss

```diff
@@ -2,2 +1,211 @@
+$cardFont: Roboto, Arial, sans-serif;
+$defaultFontWeight: 400;
+$accentColor: #060b35;
+$secondaryColor: #616070;
+$moyoBlue: #00acdc;
+$transitionTime: 300ms;
+
+html {
+  font-size: 12px;
+  font-weight: 500;
+  text-align: left;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
This is a code style issue. It's recommended to avoid using the universal selector (*) for setting properties like padding, margin, and box-sizing. It can lead to unexpected results and performance issues. Consider applying these properties to specific elements as needed.